recipes/vectorclass/all/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder/instrset_detect.cpp)
+
+set(HEADERS
```

Comment:
Maybe use glob in this context? It can make the CMakeList more portable across versions.